QuizApp/QuizApp/CustomInputFieldType.swift

```diff
@@ -0,0 +1,15 @@
+public enum CustomInputFieldType {
```

Comment:
Do not use public visibility modifier, unless you really need to make something public. By default, everything is visible inside the same module unless private/fileprivate.